gcovr/tests/add_coverages/Makefile

```diff
@@ -8,15 +8,15 @@ all:
 	$(CXX) $(CFLAGS) -DBAR -c main.cpp -o main_bar.o
 	$(CXX) $(CFLAGS) main_bar.o bar.o -o testcase_bar
 
-run: txt xml html sonarqube json
+run: txt xml html sonarqube json json_summary
 
 json_foo:
 	./testcase_foo
-	$(GCOVR) -d --json-pretty -o coverage_foo.json
+	$(GCOVR) -d --json --json-pretty -o coverage_foo.json
 
 json_bar:
 	./testcase_bar
-	$(GCOVR) -d --json coverage_bar.json
+	$(GCOVR) -d --json -o coverage_bar.json
```

Comment:
It's turned out in this commit 3aa4a34c7fe78ca5081df9e2951a4450e89f3192 output should work even without -o, so maybe it is better to leave at least one test to check correct behaviour for this issue?

Reply:
I touched that file in the html-rewrite branch (now merged). The different JSON reports in that Makefile now exercise different -o/--json combinations:

https://github.com/gcovr/gcovr/blob/1f17d4427c301f2a2a8bad50991f161a0e45aa5c/gcovr/tests/add_coverages/Makefile#L13-L23